prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -147,6 +149,9 @@ ssize_t rxr_pkt_send_data_desc(struct rxr_ep *ep,
 
 	ssize_t ret;
 
+	iov_index = tx_entry->iov_index;
```

Comment:
The changes in this file seem like a separate bug and there's also no context about this in the commit message.

Reply:
I did this because if we fail to send the packet, we need to reset `tx_entry->iov_index` and `tx_entry->iov_offset`. We might call `rxr_pkt_send_data_desc ` on a same packet for multiple times.

Reply:
Right, but that's not in the commit message. It also seems like it should be a separate bugfix at first glance.

Reply:
Yeah. It is a separate bugfix which is necessary for this patch

Reply:
In this case, you need to include this change in a separate commit with proper commit message.

Reply:
OK. I will do that.